good_egg/models.py

```diff
@@ -1,3 +1,40 @@
 from django.db import models
 
 # Create your models here.
+class Incident(models.Model):
+  YES = 'y'
+  NO = 'n'
+  yes_or_no = (
+    (YES, 'yes'),
+    (NO, 'no'),
+  )
+  category = models.CharField(max_length=100)
+  officers =  models.ForeignKey(Officer, on_delete=models.CASCADE, related_name='officer')
+  date = models.CharField(max_length=100)
+  time = models.CharField(max_length=100)
+  location = models.CharField(max_length=100)
+  formal_complaint = models.CharField(max_length=100)
+  formal_complaint_number = models.CharField(max_length=100, choices=yes_or_no, default=NO)
+  witnesses_present = models.CharField(max_length=100, choices=yes_or_no, default=NO)
+  witnesses_information = models.CharField(max_length=100)
+  User_id = models.CharField(max_length=100)
+  private = models.CharField(max_length=100, choices=yes_or_no, default=NO)
```

Comment:
Not saying you should change but why not just use `BooleanField `?

Reply:
I think I was just on a role.  I can do boolean field instead forsure.